def.bzl

```diff
@@ -0,0 +1,27 @@
+def _sha256sum_impl(ctx):
+  ctx.actions.run_shell(
+    outputs = [ctx.outputs.sha256],
+    inputs = [ctx.file.src],
+    command = "CWD=$PWD && cd `dirname {0}` && sha256sum `basename {0}` > $CWD/{1}".format(ctx.file.src.path, ctx.outputs.sha256.path),
```

Comment:
We typically build releases on MacOS, which uses a different tool (shasum -a 256)